src/main/kotlin/no/nav/familie/tilbake/behandling/BehandlingService.kt

```diff
@@ -623,6 +627,15 @@ class BehandlingService(
         }
     }
 
+    private fun kanSetteBehandlingTilbakeTilFakta(
+        behandling: Behandling,
+        behandlerRolle: Behandlerrolle,
+    ) = behandlingKanSettesTilbakeTilFakta(behandling) &&
+        ContextService.hentSaksbehandler() == behandling.ansvarligSaksbehandler &&
+        (behandlerRolle == Behandlerrolle.SAKSBEHANDLER || behandlerRolle == Behandlerrolle.BESLUTTER)
```

Comment:
Burde det vært en sjekk på om saksbehandler har forvaltningsrolle her også?

Reply:
Saksbehandler har ikke forvaltningsrolle...
Vi ønsker å åpne for enten:
- Forvaltningsrolle
- Saksbehandlerrolle og eier oppgaven

Reply:
Men gjør ikke denne metoden at en med forvaltningsrolle ikke får satt behandling tilbake til fakta?

Reply:
...+ behandling UTREDES og behandling er ikke på vent

Løser også denne favrooppgaven: 
https://favro.com/organization/98c34fb974ce445eac854de0/a64c6aad9b0d61ef6c0290bd?card=NAV-21974

Reply:
Kan tenke at mapping til behandlingDto (tilgang i UI aksjon er ikke er for forvaltere) - forvaltere kan jo bruke forvaltningsrutine som før?

Reply:
Åja, selvfølgelig. Skjønner, takk for oppklaring!

Reply:
Men takk for at du sa ifra - det er noe her som er smelly - jeg fortsetter å se på det!